doc/ip/spi.md

```diff
@@ -12,15 +12,17 @@ In Sonata, there are multiple uses for SPI:
 - Arduino shield
 - mikroBUS click
 
-The offset for each of the blocks is shown below, with each additional block having a `0x1000` offset from the previous.
-
-| SPI Instance          | Offset (from SPI base) |
-|-----------------------|------------------------|
-| LCD Screen            | 0x0000                 |
-| Ethernet MAC          | 0x1000                 |
-| SPI0                  | 0x2000                 |
-| SPI1                  | 0x3000                 |
-| SPI2                  | 0x4000                 |
+The offset for each of the blocks is shown below, with each additional block having a 0x1000 offset from the previous.
+The offsets in the table below are from the SPI base address, which is 0x8030_0000.
+The table also shows the connections of the chip select (CS) wires, some of which are used for block specific functions like reset and data/control signals.
+
+| Name         | Offset | Blocks                    | CS0          | CS1          | CS2          | CS3      |
+|--------------|--------|---------------------------|--------------|--------------|--------------|----------|
+| LCD Screen   | 0x0000 | *Only* LCD                | CS           | Data/control | Reset        |          |
+| Ethernet MAC | 0x1000 | *Only* Ethernet           | CS           | Reset        |              |          |
+| SPI0         | 0x2000 | Flash or MicroSD          | Flash        | MicroSD      |              |          |
+| SPI1         | 0x3000 | HAT SPI0, Shield, Pmod0   | HAT or Pmod0 | HAT or Pmod0 | Pmod0        | Shield   |
+| SPI2         | 0x4000 | HAT SPI1, mikroBUS, Pmod1 | HAT or Pmod1 | HAT or Pmod1 | HAT or Pmod1 | mikroBUS |
 
 Please refer to the [pin multiplexer](pinmux/pin-mappings.md) on how to connect SPI 0, 1 and 2.
```

Comment:
Can you also update this line to add a link to the Pinmux IP doc page as well as the mapping diagram, as right now it shows you what connections can be made but not how to connect it.

Reply:
Good idea, but unfortunately we currently cannot link to the main pinmux page because links to README.md get replaced with README.html not index.html. So I'll leave this for now until that's fixed.